src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerChangeService.java

```diff
@@ -75,12 +73,12 @@ public List<Change> collateBeneficialOwnerChanges(
    * related to the beneficial owners of 'Individual' type. If no changes are detected, an empty
    * list is returned.
    */
-  private List<Change> individualBeneficialOwnerChange() {
+  private List<Change> individualBeneficialOwnerChange(Map<String, Object> logMap) {
     var beneficialOwnersIndividual = overseasEntitySubmissionDto.getBeneficialOwnersIndividual();
     return beneficialOwnersIndividual
         .stream()
         .filter(bo -> bo.getChipsReference() != null)
-        .map(this::covertBeneficialOwnerIndividualToChange)
+        .map(individualBO -> covertBeneficialOwnerIndividualToChange(individualBO, logMap))
```

Comment:
Would you mind doing a search on the whole file for the word COVERT and change them to CONVERT? Cheers.